checker/src/main/java/org/checkerframework/checker/signedness/SignednessVisitor.java

```diff
@@ -301,23 +305,27 @@ public Void visitBinary(BinaryTree node, Void p) {
         switch (kind) {
             case DIVIDE:
             case REMAINDER:
-                if (leftOpType.hasAnnotation(Unsigned.class)) {
+                if (leftOpType.hasAnnotation(Unsigned.class)
```

Comment:
Instead of changing every use of `atm.hasAnnotation(Unsigned.class)` to  `atm.hasAnnotation(Unsigned.class) || atm.hasAnnotation(PolyUnsigned.class)`, create a method that does that and call it multiple times.

Reply:
Added two methods, one for unsigned and second for signed.